.github/pull_request_template.md

```diff
@@ -26,9 +31,4 @@ If included, identify any third party code and provide text file of license
 
 **Contributor Info - All information REQUIRED for consideration of pull request**
 Full name and company/organization/center of all contributors ("Personal" if individual work)
-- If NASA Civil Servant Employee or GSFC Contractor on SES II
-  - Address/email/phone and contract/task information (if applicable) must be on file
-- Else if Company
-  - **HAND SIGNED** Company CLA must be on file (once per release): [Company CLA](https://github.com/nasa/cFE/blob/master/docs/GSC_18128_Corp_CLA_form_1219.pdf)
-- Else if Individual
-  - **HAND SIGNED** Individual CLA must be on file (once per release): [Individual CLA](https://github.com/nasa/cFE/blob/master/docs/GSC_18128_Ind_CLA_form_1219.pdf)
+ - Note CLA's apply to software contributions.
```

Comment:
no apostrophe `'`, should add the word "only" before software contributions

Reply:
@astrogeco - note that when I tried using the link to the subsection in the cFS contributing guide it didn't go to the appropriate section.  Probably want to confirm this actually works...

Reply:
It won't work until nasa/cFS#448 is merged. It does work on the fork https://github.com/chillfig/cFS/blob/add_cla/CONTRIBUTING.md#contributor-license-agreement-cla